src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/UpdateVCFSequenceDictionaryIntegrationTest.java

```diff
@@ -89,8 +93,9 @@ private void testBadUpdateSequenceDictionary(
             final File inputSourceFile,
             final File inputReferenceFile,
             final String masterSequenceDictionary,
-            final boolean replace) {
-        updateSequenceDictionary(inputVariantsFile, inputSourceFile, inputReferenceFile, masterSequenceDictionary, replace);
+            final boolean replace,
+            final boolean dontValidate) {
```

Comment:
Change the name of this arg to "disableSequenceDictionaryValidation" (here and in all of the other test methods).